### ISSUE TITLE
Smart GT06 on Traccar

### ISSUE THREAD

**Issue by itji** [traccar/traccar#100]
Hi, I use Chinesse made Smart GT06 for the tracker, try to set it up on Traccar, set on 5000 port, but nothing happen, i have looked on the log file still empty (0KB) any idea? TIA

**Comment by aminedeghais** [comment]
Hi everyone,
have any one of you get traccar operated with GT06 in java ?

**Comment by tananaev** [comment]
Many people reported that GT06 works with Traccar.

**Comment by dev-jucelio** [comment]
My GT06 is also reporting
DQP ♥ PEG ☺╩╧
DQP ♥ PEG ☻ ° T
DQP ♥ PEG ☺╩╧

how do I decrypt it C ++?

**Comment by tananaev** [comment]
@dev-jucelio, please provide messages in HEX format.

**Comment by dev1rivastecnologia** [comment]
The more they are in this format

**Comment by dev1rivastecnologia** [comment]
How do I turn into HEX? GT06 Accurate

**Comment by tananaev** [comment]
@dev1rivastecnologia, it depends on how you get binary data.

**Comment by dev1rivastecnologia** [comment]
The data are not coming in binary format is in ASCII

**Comment by tananaev** [comment]
No it's not. It's clearly binary data.

**Comment by dev1rivastecnologia** [comment]
So I'm turning the wrong way?

**Comment by tananaev** [comment]
@dev1rivastecnologia, GT06 only supports binary format as far as I know. If you want to continue discussion, please create a separate thread.

**Comment by ravi823** [comment]
I am using GT06N by Concox, but getting GPS jumps can anyone help me with that?? i have put the parameter as TIMER,10,10# and SENDS,5#. are they correct or Is there any specific settings for more accurate tracking?

**Comment by karthikbalu** [comment]
So can anyone tell me if traccar supports GT06 now? i tried with the latest version but doesnt work also GT06 protocol manual http://www.iconcox.com/uploads/soft/130627/1-13062F94158.pdf says "device sends GPS data only after login message is acknowledged by the server" so does traccar return ACK message back to device?  Thanks

**Comment by tananaev** [comment]
Traccar supported GT06 protocol for a very very log time now. If it doesn't work for you, you are doing something wrong. Traccar does return ACK. Please don't comment on an ancient thread. If something doesn't work, create a new issue.

**Comment by ghusousa** [comment]
em que arquivo coloco o protocolo gt06?

**Comment by eze27** [comment]
> @tananaev Hi, i'm using qbit gps concox on my own server but i have a trouble with data i'm receiving from gps, first of all not sure what protocol to use, i'm trying gt02e and i can login but not next step that is heartbeat because server needs protocol number 13 however it gets 23.

**Comment by tananaev** [comment]
Please don't comment on old closed threads.

**Comment by mukesh008** [comment]
I want to build own server  in C# how to data from this Devices gt06 Chinese Model
